ansible/roles/screenly/files/screenly.conf

```diff
@@ -15,6 +15,9 @@ audio_output = hdmi
 ; Shuffle playlist before playing (otherwise sorted by name)
 shuffle_playlist = False
 
+; Stream asset duration default value
+default_streaming_duration = 300
```

Comment:
Yeah, I assumed it was in seconds, but let's just add that for clarity. Just add "(seconds)" to the end of the sentence. Do the same for `default_duration`.